doc/manual/src/command-ref/conf-file-prefix.md

```diff
@@ -17,7 +17,7 @@ By default Nix reads settings from the following places:
 
     Otherwise it will look for `nix/nix.conf` files in `XDG_CONFIG_DIRS`
```

Comment:
This does not actually fully match the [basedir-spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html):

> If $XDG_CONFIG_DIRS is either not set or empty, a value equal to /etc/xdg should be used.

We would need `/etc/xdg` as a fallback in

https://github.com/NixOS/nix/blob/c5fd0b46ae56447d9d0a2028d464a5bbf753bf55/src/libutil/util.cc#L565